kubeprod/cmd/install.go

```diff
@@ -31,8 +31,8 @@ import (
 )
 
 const (
-	FlagManifests       = "manifests"
-	DefaultManifestBase = "https://github.com/bitnami/kube-prod-runtime/manifests/"
+	FlagManifests          = "manifests"
+	defaultManifestBaseFmt = "https://github.com/bitnami/kube-prod-runtime/raw/%s/manifests/"
```

Comment:
While testing the PR i found the right gitTag is not applied  and always appears to be set to `master` (default) instead of the one configured at build time.

Reply:
The first iteration of this PR had that issue.  Please retry with something >= 9baf12cff448a0985901156482a6e13fc8138f58

Reply:
gah, I get the same thing as you too.  I thought once I'd fixed the default shown in `--help` that that would propagate through the rest, but apparently not :(

Reply:
ok, fixed for real this time.  PTAL (the only change is `cmd/root.go:UpdateFlagDefaults()`)